src/store.ts

```diff
@@ -18,12 +18,17 @@ export enum PerformanceMeasurement {
   All = "all"
 }
 
+export interface DevToolOptions {

```

Comment:
let's call it in plural `DevToolsOptions`

Reply:
keep in mind this would need to be updated in all places, also the docs

Reply:
actually, the proper interface for this can be looked up [here](https://github.com/zalmoxisus/redux-devtools-extension/blob/master/npm-package/index.d.ts#L27-L45)